zerver/webhooks/jira/doc.md

```diff
@@ -14,8 +14,12 @@ Server versions 5.2 or greater. For older versions, you'll need our
 
 1. {!generate-webhook-url-basic.md!}
 
-1. Go to your Jira **Site administration** page, and click on **Jira**.
-   In the left sidebar, scroll down, and under **Advanced**, click
+1. Go to your Jira **Site administration** page. Click on the
+   **dropdown**( <i class="fa fa-ellipsis-h" aria-hidden="true"></i> )
+   under the **Actions** column on your **Jira** app, and select
+   the **Jira Settings** option.
+
+1. In the left sidebar, scroll down, and under **Advanced**, click
    **WebHooks**. Click **+ Create a WebHook**.
```

Comment:
I'd combine this with the previous step. It's a bit long, but our general structure for these documentation pages is one step with all navigation instructions then one step with configuration instructions, and repeat as necessary.